app/src/main/java/com/example/firstandroid/presentation/add_bathroom/NewBathroomScreen.kt

```diff
@@ -84,7 +84,7 @@ fun AddBathroomDialog(
             Button(
                 onClick = {
                     scope.launch {
```

Comment:
This still doesn't seem necessary. The viewmodel is the class that holds the state. So you do not need to pass it the state.